### PR TITLE
Added Survey Manager and SRO Roles

### PR DESCRIPTION
**Pull request by elthorne**

### CODE REVIEW

**Comment by elthorne** [comment]
This PR adds the following roles with subsequent features and access role management:

* Senior Research Officer. Can view most pages. Can edit Researcher roles only.
* Survey Manager. Can view most pages. Can edit Senior Research Officer and Researcher roles only.

See LU-7706 & LU-7707 fore more info.

Manually completed the following tests in https://dev-el-35-bsm.social-surveys.gcp.onsdigital.uk/:

As a DST_Tech role I can:
* Add/Edit Senior Research Officer (SRO) and Survey Manager roles

As a BDSS role I can:
* Add/Edit Senior Research Officer (SRO) and Survey Manager roles

As an SRO role I can:
* View the Interviewing, Sample file, Case Summary, CATI dashboard, Manage Users, Manage Surveys and Manage Field Periods pages
* Add/Edit Researcher roles only

As an Survey Manager role I can:
* View the Interviewing, Sample file, Case Summary, CATI dashboard, Manage Users, Manage Surveys and Manage Field Periods pages
* Add/Edit SRO and Researcher roles only

File: alembic/versions/d04295661a60_003_add_new_roles.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+                   '"manage_field_periods", "manage_users"'
+    sro_manage_users = '["Research"]'
+
+    op.execute(
+        "INSERT INTO role VALUES('Senior_Research_Officer', 'Senior Research Officer', '" + sro_features + "', '" + sro_manage_users + "')")
+
+    # Survey Manager role
+    survey_manager_features = '"interviewing", "sample_upload", "case_summary", "cati_dashboard", ' \
+                              '"manage_surveys", "manage_field_periods", "manage_users"'
+    survey_manager_manage_users = '["Survey_Manager", "Research"]'
```
Comment (matthewandrewpalmer) [inline alembic/versions/d04295661a60_003_add_new_roles.py:32]: Survey managers should not be able to able to manage Survey managers.
But should be able to manage Senior_Research_Officer's

```suggestion
    survey_manager_manage_users = '["Senior_Research_Officer", "Research"]'
```

**Review by matthewandrewpalmer — CHANGES_REQUESTED** [review]
Working fine in sandbox, tested syncing to Blaise and that worked fine. One change suggested around user management.

**Review by matthewandrewpalmer — APPROVED** [review]
🦆 Looks good.

**Pull request merged**